### PR TITLE
New adopter: despegar.com

### PR DESCRIPTION
**Pull request by tarossi**

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ URLs if possible) and [submit a pull request][1]!
 * [Chartboost](https://www.chartboost.com/)
 * [Crimson Hexagon](http://www.crimsonhexagon.com/)
 * [DICE](http://dice.se/)
+* [Despegar](http://www.despegar.com)
```
Comment (mosesn) [inline ADOPTERS.md]: for consistency, could you change this to http://www.despegar.com/?

Comment (mosesn) [inline ADOPTERS.md]: also, could you put this line about the DICE line so we can stay in alphabetic order?

**Comment by mosesn** [comment]
LGTM except for a couple nits

**Comment by travisbrown** [comment]
Thanks, @tarossi! I've pulled this in to be merged.

**Comment by travisbrown** [comment]
Merged in 1dc81418793fdb6a122bfd36ec7fbf3a9facb333, thanks!

**Pull request closed without merging**